vllm/v1/worker/gpu_model_runner.py

```diff
@@ -2467,7 +2467,7 @@ def propose_draft_token_ids(sampled_token_ids):
             effective_drafter_max_model_len = (
                 self.speculative_config.draft_model_config.max_model_len)
         input_fits_in_drafter = spec_decode_common_attn_metadata and (
-            spec_decode_common_attn_metadata.seq_lens.max() +
+            spec_decode_common_attn_metadata.seq_lens_cpu.max() +
```

Comment:
Is it possible to use `.max_seq_len` here?

Reply:
Indeed, and it is simpler!